mithril-common/src/cardano_block_scanner/chain_reader_block_streamer.rs

```diff
@@ -111,7 +111,7 @@ impl ChainReaderBlockStreamer {
             from,
             until,
             max_roll_forwards_per_poll,
-            logger,
+            logger: logger.new(slog::o!("src" => "ChainReaderBlockStreamer")),
```

Comment:
To avoid to repeat the struct name (that is a common mistake), it's possible to use `std::any::type_name::<Self>()`.
The name is prefixed by the module name so, we have to remove it if we want only the name.
 We can add a `.split("::").last().unwrap()` for example even it's probably not the best solution

Reply:
Done using an extension trait :).